src/kibana/plugins/discover/components/field_chooser/field_chooser.js

```diff
@@ -133,7 +133,7 @@ define(function (require) {
               schema: 'segment',
               params: {
                 field: field.name,
-                interval: 'auto'
+                interval: 'hour'
```

Comment:
Hmm, ok. I figured and was thinking about this last night.

Ideally we look at the amount of time between start and end and find a
reasonable interval there.

The only question is, what is a good interval for a given gap in time.
On Mar 25, 2015 7:24 AM, "Spencer" <notifications@github.com> wrote:

> In src/kibana/plugins/discover/components/field_chooser/field_chooser.js
> <https://github.com/elastic/kibana/pull/3443#discussion_r27124564>:
>
> > @@ -133,7 +133,7 @@ define(function (require) {
> >                schema: 'segment',
> >                params: {
> >                  field: field.name,
> > -                interval: 'auto'
> > +                interval: 'hour'
>
> We can't just set the interval to 'hour' for every request man.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/elastic/kibana/pull/3443/files#r27124564>.
>